templates/taxi/driver_list.html

```diff
@@ -5,10 +5,17 @@ <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver_detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
-        </li>
+        {% if driver.id == user.id %}
+          <li>
+            <a href="{% url "taxi:driver_detail" pk=driver.id %}">{{ driver.username }}</a>
+            (Me)
```

Comment:
You can add "(me)" part in the if statement after the link, so you don't need to repeat "a href.." twice